packages/react-components/react-combobox/docs/Spec.md

```diff
@@ -2,17 +2,19 @@
 
 ## Background
 
-The basic purpose of a Combobox is to allow users to select one or more values from among a set of options. The semantics and behavior are roughly similar to a more complex version of an HTML `<select>` element, with more functionality and styling control.
+The basic purpose of a Combobox or Dropdown is to allow users to select one or more values from among a set of options. The semantics and behavior are roughly similar to a more complex version of an HTML `<select>` element, with more functionality and styling control.
 
-A combobox can be single or multi-select, and it can be editable or select-only. More options are covered in the [Variants section](#variants).
+A combobox can be single or multi-select, and it can be editable (`<Combobox>`) or select-only (`<Dropdown>`). More options are covered in the [Variants section](#variants).
 
-The basic structure of a Combobox has two pieces: the faceplate, which is always rendered on the page, and displays the current selection, and the popup Listbox, which contains the set of options.
+The basic structure of a Combobox or Dropdown has two pieces: the faceplate, which is always rendered on the page, and displays the current selection, and the popup that contains a list of options.
 
-The Listbox can also be used as a standalone inline selection widget, similar to a `<select size="n > 1">` or `<select multiple>`. The Listbox has the same options and functionality whether used on its own, or within a Combobox.
+### Combobox vs. Dropdown
 
```

Comment:
Looks like this was cut off.

Reply:
Listbox isn't provided as a standalone widget anymore, so I removed that section from the docs

Reply:
oh, haha, didn't realize the sentence below was cut off 😂